Bottom: ca66658d888991d907cf49b521cd24aa46035893
Top:    700a05dda2bf0cded7088b8185d23182fc3a6801
Author: Sergey Alyoshin <alyoshin.s@gmail.com>
Date:   2010-05-26 23:21:28 +0100

Subject: PCB localization

Add top level menu to localization


---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 1511e09..f9e425a 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -46,7 +46,7 @@ MainMenu =
 # File Menu
 #
 
-  {File
+  {"File"
    {"Save layout" Save(Layout) tip="Saves current layout"}
    {"Save layout as..." Save(LayoutAs) tip="Saves current layout into a new file"}
    -
@@ -80,7 +80,7 @@ MainMenu =
 # Edit Menu
 #
 
-  {Edit 
+  {"Edit"
    {"Undo last operation" Undo() a={"U" "<Key>u"}}
    {"Redo last undone operation" Redo() a={"Shift-R" "Shift<Key>r"}}
    {"Clear undo-buffer" Undo(ClearList) a={"Shift-Ctrl-U" "Shift Ctrl<Key>u"}}
@@ -119,7 +119,7 @@ MainMenu =
 # View Menu
 #
 
-  {View 
+  {"View"
    {"Enable visible grid" checked=drawgrid Display(Grid)}
    {"Grid units"
     {"mil" checked=grid_units_mm,0 SetUnits(mil)}
@@ -197,8 +197,8 @@ MainMenu =
 
 #
 # Settings menu
-#  
-  {Settings
+#
+  {"Settings"
    {"'All-direction' lines" checked=alldirection Display(Toggle45Degree) a={"." "<Key>."}}
    {"Auto swap line start angle" checked=swapstartdir Display(ToggleStartDirection)}
    {"Orthogonal moves" checked=orthomove Display(ToggleOrthoMove)}
@@ -221,11 +221,11 @@ MainMenu =
    -
    {"Vendor drill mapping" ToggleVendor() checked=VendorMapOn}
   }
-    
+
 #
 # Select menu
 #
-  {Select 
+  {"Select"
    {"Select all visible objects" Select(All)}
    {"Select all connected objects" Select(Connection)}
    -
@@ -251,7 +251,7 @@ MainMenu =
    -
    {"Optimize selected rats" DeleteRats(SelectedRats) AddRats(SelectedRats)}
    {"Auto-route selected rats" AutoRoute(SelectedRats) a={"Alt-R" "Alt<Key>r"}}
-   {"Rip-up selected auto-routed tracks" RipUp(Selected)}
+   {"Rip up selected auto-routed tracks" RipUp(Selected)}
    -
    {"Change size of selected objects"
     {"Lines -10 mil" ChangeSize(SelectedLines,-10,mil) ChangeSize(SelectedArcs,-10,mil)}
@@ -282,7 +282,7 @@ MainMenu =
 #
 # Buffer menu
 #
-  {Buffer
+  {"Buffer"
    {"Cut selection to buffer" GetXY(Click to set the snap point for this buffer)
     PasteBuffer(Clear) PasteBuffer(AddSelected) RemoveSelected() Mode(PasteBuffer)}
    {"Paste buffer to layout" Mode(PasteBuffer)}
@@ -306,11 +306,11 @@ MainMenu =
    {"#4" checked=buffer,4 PasteBuffer(4) a={"Shift-4" "Shift<Key>4"}}
    {"#5" checked=buffer,5 PasteBuffer(5) a={"Shift-5" "Shift<Key>5"}}
   }
-    
+
 #
 # Connects menu
 #
-  {Connects 
+  {"Connects"
    {"Lookup connection to object" GetXY(Click on the object) Connection(Find) a={"Ctrl-F" "Ctrl<Key>f"}}
    {"Reset scanned pads/pins/vias" Connection(ResetPinsViasAndPads) Display(Redraw)}
    {"Reset scanned lines/polygons" Connection(ResetLinesAndPolygons) Display(Redraw)}
@@ -348,11 +348,11 @@ MainMenu =
    -
    {"Apply vendor drill mapping" ApplyVendor()}
   }
-    
+
 #
 # Info Menu
 #
-  {Info
+  {"Info"
    {"Generate object report" ReportObject() a={"Ctrl-R" "Ctrl<Key>r"}}
    {"Generate drill summary" Report(DrillReport)}
    {"Report found pins/pads" Report(FoundPins)}
@@ -458,7 +458,7 @@ MainMenu =
 #
 # Window Menu
 #
-  {Window
+  {"Window"
    {"Library" DoWindows(Library) a={"i" "<Key>i"}}
    {"Message Log" DoWindows(Log)}
    {"DRC Check" DoWindows(DRC)}
@@ -477,14 +477,14 @@ PopupMenus =
       {"Operations on selections"
        {"Unselect all objects" Unselect(All)}
        {"Remove selected objects" RemoveSelected()}
-       {"Copy selection to buffer" 
-	GetXY(Click to set the snap point for this buffer) 
+       {"Copy selection to buffer"
+	GetXY(Click to set the snap point for this buffer)
 	PasteBuffer(Clear)
 	PasteBuffer(AddSelected)
 	Mode(PasteBuffer)
        }
        {"Cut selection to buffer"
-	GetXY(Click to set the snap point for this buffer) 
+	GetXY(Click to set the snap point for this buffer)
 	PasteBuffer(Clear)
 	PasteBuffer(AddSelected)
 	RemoveSelected()
@@ -493,7 +493,7 @@ PopupMenus =
        {"Convert selection to element" Select(Convert)}
        {"Auto place selected elements" AutoPlaceSelected()}
        {"Autoroute selected elements" AutoRoute(SelectedRats)}
-       {"Rip up selected auto routed tracks" RipUp(Selected)}
+       {"Rip up selected auto-routed tracks" RipUp(Selected)}
       }
       {"Operations on this location"
        {"Generate object report" GetXY(Click on the object) Report(Object)}
